todo_app/app.py

```diff
@@ -48,17 +60,20 @@ def add_item():
         todo_item_status = request.form.get('status')
         # mongo.addItemstoMongo({'cardid':todo_item_id})
         mongo.addItemstoMongo(todo_item_id, todo_item_title, todo_item_status)
+        app.logger.info("added items to Mongo")
```

Comment:
This message could be made a lot more useful. Why not include the item id? And the id of the current user?